README.md

```diff
@@ -1,8 +1,6 @@
 <a name="readme-top"></a>
 
-<div align="center">
-  <img src="nahnah_logo.png" alt="logo" width="250"  height="auto" />
-</div>
+
 
 <!-- TABLE OF CONTENTS -->
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update the `Getting Started` section content with correct text content as the current ones seems to be wrong according to your project infos.
![Screenshot from 2023-01-31 10-53-43](https://user-images.githubusercontent.com/83042742/215714230-7819da27-9b7a-42d3-b483-617801775270.png)